LukeMeilerHW8P1.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>That is not what you actually plot. But I assume you have it a t one point.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>